openapi.yaml

```
@@ -0,0 +1,2427 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+---
+openapi: 3.0.3
+
+info:
+  title: "Airflow API (Stable)"
+  description: Apache Airflow management API.
+  version: '1.0.0'
+  license:
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
+  contact:
+    name: Apache Foundation
```

```suggestion
    name: Apache Software Foundation
```

Although maybe we should change it to Airflow PMC & Committers or something, I am not sure

---

Airflow PMC and commiters are not a legal entity. It seems to me that it's better to provide the name of an entity that is easy to find.  However, I am not sure either.

---

Related commit: https://github.com/apache/airflow/pull/8721/commits/abef5b999da5e75d81e488228bda51bdd5827ccf